<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendsGUI</class>
 <widget class="QWidget" name="FriendsGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Friend list:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="friendList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="viewProfile">
         <property name="text">
          <string>View profile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeFriend">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Enter username to add friend:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="friendNameInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="addFriend">
       <property name="text">
        <string>Add friend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="message">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
